README.md

```diff
@@ -43,7 +43,7 @@ information on getting started using and developing with the fabric, SDK and cha
 - [v1.2](http://hyperledger-fabric.readthedocs.io/en/release-1.2/)
 - [v1.1](http://hyperledger-fabric.readthedocs.io/en/release-1.1/)
 - [v1.0](http://hyperledger-fabric.readthedocs.io/en/release-1.0/)
-- [master branch (development)](http://hyperledger-fabric.readthedocs.io/en/master/)
+- [master branch (development)](http://hyperledger-fabric.readthedocs.io/en/latest/)
```

Comment:
This suggested update is correct because the master branch development points to the latest branch of the docs : 
`master branch (development)](http://hyperledger-fabric.readthedocs.io/en/latest/)`